pkg/projects/provider_test.go

```diff
@@ -47,11 +39,16 @@ func getTestResty(t *testing.T) *resty.Client {
 
 func testAccPreCheck(t *testing.T) {
 	restyClient := getTestResty(t)
-	// TODO check the payload and make sure it's the right license type
-	_, err := restyClient.R().Get("/artifactory/api/system/licenses/")
+	resp, err := restyClient.R().Get("/artifactory/api/system/licenses/")
 	if err != nil {
 		t.Fatal(err)
 	}
+
+	body := fmt.Sprintf("%s", resp.Body())
```

Comment:
So, I think if you don't give it a result definition, it's a `map[string]interface{}` - which means you could can do map traversal.

Reply:
I kind of feel like this logic should move into the execution line - I mean, I want my TF to tell me right away if it isn't going to work. 

Can you move this code to RT as well?

Reply:
I'll do you one better. I create `License` struct and get the body marshalled 😄 

I'll create a ticket for this for RT